modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -184,6 +184,10 @@ Object Store v2 uses TLS for secure transport. Data at rest is stored using FIPS
 Object Store v2 performs I/O for each read and write. With Object Store v2, the API call is localized to the
 same data center as the Runtime Manager app.
 
+== What happens when OSv2 option is changed from disabled to enabled status?
```

Comment:
```suggestion
== What happens when you enable the *Use Object Store v2* option?
```

Reply:
Hi hannanelson, the suggested change "What happens when you enable the *Use Object Store v2* option?" is not clear as the option is enabled by default. The Q/A IS more about the output of re-enabling the osv2 option where the osv2 option is changed from un-ticked to re-ticked status.

Reply:
@hannanelson , Since this is only for the scenario where the customer enables->disables->enables, should we change it to reflect this ?

Reply:
@yunmei-zhang , @schinthakindims - clarified